src/components/ebay-chip/index.marko

```diff
@@ -0,0 +1,19 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+$ const { renderBody, deleteButtonLabel, ...htmlAttributes } = input;
+
+<span class="chip" ...processHtmlAttributes(htmlAttributes)>
+    <span class="chip__text" id:scoped="text">
+        <${input.renderBody}/>
+    </span>
+    <if(input.deleteButtonLabel)>
+        <button
+            type="button"
+            class="chip__button"
+            aria-label=input.deleteButtonLabel
```

Comment:
We usually prefix aria-labels input variables with `a11y` something. So maybe call this `a11yDeleteButton`

Reply:
Thanks for the consistency